Assets/BowAndArrowImproved/Scripts/BAAIEnemy.cs

```diff
@@ -9,6 +10,17 @@ public class BAAIEnemy : MonoBehaviour,BAAIIDeathable
     public UnityEvent<GameObject> onDeath => _onDeath;
 
     [SerializeField] private List<AudioClip> breakClips;
+    [SerializeField] private List<AudioClip> warCryClips;
+
+    private void Start()
+    {
+        PlayWarCry();
+    }
+
+    private void PlayWarCry()
+    {
+        AudioSource.PlayClipAtPoint(warCryClips[Random.Range(0, warCryClips.Count)], transform.position);
+    }
 

```

Comment:
do it in onEnable so if we enable object pooling on enemies this will still work